### ISSUE TITLE
Add `context` to `addTelemetryInitializer` callback?

### ISSUE THREAD

**Issue by dotansimha** [microsoft/ApplicationInsights-JS#1026]
In https://github.com/microsoft/ApplicationInsights-JS , you can modify the telemetry using `addTelemetryProcessor` which executed with `envelop` and `context` (the context changes according to the collected data, for example: the req/res for HTTP requests). 

Any chance we can do the same for the library? :)

**Comment by dotansimha** [comment]
Nevermind, needed to pust this under https://github.com/microsoft/ApplicationInsights-JS#readme :D

**Comment by markwolff** [comment]
Yep, ideally this SDK should share, as much as possible, an API/feature surface with the node.js SDK. I'll add this to our backlog

**Comment by dotansimha** [comment]
Thanks @markwolff ! I can create a PR to add it, can you maybe just point me to the code that manages that?

**Comment by markwolff** [comment]
@dotansimha It would be inside the "analytics plugin". Here is the entry point (you will also need to update the same function located in `AISKU/Initialization.ts` to match the interface change.
https://github.com/microsoft/ApplicationInsights-JS/blob/9705984134fcc5244bc5f6838163d766a8be5744/vNext/extensions/applicationinsights-analytics-js/src/JavaScriptSDK/ApplicationInsights.ts#L455-L457

This plugin already has "context" available to it via `this._properties.context`.
https://github.com/microsoft/ApplicationInsights-JS/tree/master/vNext/extensions/applicationinsights-properties-js/src/Context
